DALK.PL_ANALYZER/Models/Matches/Person.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace DALK.PL_ANALYZER.Models.Matches
+{
+    public class Person
```

Comment:
If this is gonna be some sort of DB model, then should be placed in different folder/namespace